kf_lib_data_ingest/etl/transform/transform.py

```diff
@@ -60,18 +64,18 @@ def _read_output(self):
 
     def _write_output(self, output):
         """
-        Write output of transform stage to file
+        Write output of transform stage to JSON file
```

Comment:
Is there any chance to also have tsv outputs for transform like with extract?

Reply:
Hmm .. yes .. maybe?......................... OKAYYYY